src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -2398,10 +2403,19 @@ public Response deactivateBannerMessage(@PathParam("id") Long id) throws Wrapped
     @Path("/bannerMessage")
     public Response getBannerMessages(@PathParam("id") Long id) throws WrappedResponse {
 
-        return ok(bannerMessageService.findAllBannerMessages().stream()
-                .map(m -> jsonObjectBuilder().add("id", m.getId()).add("displayValue", m.getDisplayValue()))
-                .collect(toJsonArray()));
+        List<BannerMessage> messagesList = bannerMessageService.findAllBannerMessages();
+
+        for (BannerMessage message : messagesList) {
+            if ("".equals(message.getDisplayValue())) {
+               return error(Response.Status.INTERNAL_SERVER_ERROR, "No banner messages found for this locale.");
```

Comment:
Should we return the locale?

Also, should we add something to the guides to explain how to prevent this error?

Reply:
I added to the docs a warning: 

![image](https://github.com/IQSS/dataverse/assets/142103991/452757a0-f358-4496-b6ca-e62fd41e5dfc)

I also added a makefile that would make it easier to build the guides by just simply typing "make guides" let me know what you think.